decisions/wc_economy_decisions.txt

```diff
@@ -0,0 +1,137 @@
+###################################
+#
+# Economy decisions
+#
+###################################
+
+# Written by Miguel "Cipher" Rodrigues, reworked by Zuzu
+
+decisions = {
+	borrow_money = {
+		ai_check_interval = 30
+		
+		only_playable = yes
+		
+		potential = {
+			NOT = { has_character_modifier = debt_360_gold }
+			
+			NOT = { has_character_flag = ongoing_borrow_money_flag }
+		}
+		
+		allow = {
+			prisoner = no
+		}
+		
+		effect = {
+			set_character_flag = ongoing_borrow_money_flag
+			
+			character_event = { id = WCECO.100 }
+		}
+		
+		revoke_allowed = {
+			always = no
+		}
+		
+		ai_will_do = {
+			factor = 1
+			modifier = {
+				factor = 0
+				wealth = 25
+				war = no
+			}
```

Comment:
I created vassal, because he has no money, he ask me for a loan.
```suggestion
			modifier = {
				factor = 0
				wealth = 0
				war = no
			}
```

Reply:
@MemeWizard fixed in 9d6a21f2739d44c747b0f4129fd64e8400f80ec5.